### DISCUSSION TITLE
Genesis read write Sram

### DISCUSSION THREAD

**Discussion by Gunnerolivier** [sanni/cartreader#728] [Need help]
Good evening,

I have issues with 2 games on Genesis with sram. Landstalker and Beyond Oasis.

Read and write functions on save the hero does not work correctly.

The dumper reads correctly my backup. When I try to write my srm file (copying srm file of my Kega emulator to my cartridges), nothing is saved anymore when I launch games.

I renamed saves like games data dump.

I asked my question to the Admin, he thought for my hardware issues. 

I tested on 2 Megadrive, a Megadrive 1 Jap, as well as a Megadrive 2 Pal. I have the same issues with 2 consoles.

Before the tests the Admin thoughts there was isssues consoles sram.

Someone success to solved this issues ?

Thank you.

**Comment by sanni** [comment]
Have you converted the emulator save file before writing it to the cartridge? https://github.com/sanni/cartreader/discussions/593

**Reply by Gunnerolivier** [reply]
Hi Sanni,

Why convert Genesis .srm files ? 

Or else how I do please ?

Thank you

**Reply by Gunnerolivier** [reply]
@sanni I I succeeded to write Sram of Story of Thor and Landstalker. You must to convert the file .srm at right size, for Story of Thor the file .srm did 10 ko from Kega, you have to go on save file converter website, advanced and resize, then you choose the 16ko size, download the file, finally you have the good file for write it on Sram.

Thank you.
